target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -34,6 +34,10 @@ gnubee,gb-pc2)
 	ucidef_set_led_switch "lan1" "lan1" "$boardname:green:lan1" "switch0" "0x01"
 	ucidef_set_led_switch "lan2" "lan2" "$boardname:green:lan2" "switch0" "0x10"
 	;;
+iptime,a6ns-m)
+	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "$boardname:blue:wlan5g" "wlan0"
+	ucidef_set_led_netdev "wlan2g" "WiFi 2.4GHz" "$boardname:blue:wlan2g" "wlan1"
```

Comment:
No trigger for DTS?

Reply:
I tested dts trigger method too, but phy0radio did not work at all, and only phy0tx worked. It seems like mt7615e does not have phy0tpt trigger. https://github.com/openwrt/openwrt/pull/2169#issuecomment-507066119